code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm

```diff
@@ -0,0 +1,72 @@
+/mob/living/simple_animal/hostile/abnormality/forsaken_murderer //designed to be a very forgiving and standard abnormality.
+	name = "forsaken murderer"
+	desc = "A unhealthy looking human bound in a full body straightjacket. His neck is broken and in the middle of his forehead is a old wound that refuses to heal."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "forsakenmurdererinert"
+	icon_living = "forsakenmurdererinert"
+	icon_dead = "forsakenmurdererdead"
+	del_on_death = FALSE
+	maxHealth = 600 //originally was 270. Fragment health is 800 with a original game health of 230 so techically forsaken murderer has more health than fragment? Ill round the numbers to 600 since 270 can be rounded to 300 and doubled.
+	health = 600
+	rapid_melee = 1 //attack speed modifier. 2 is twice the normal.
+	move_to_delay = 5 //speed? lower is faster.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	melee_damage_lower = 6 //the lowest damage in regular attacks. Normal murderer is 2~4 so we double it.
+	melee_damage_upper = 12 // fragments Lobotomy Corp damage was 3~4 so im giving murderer a larger gap between his lower and upper damage. Unsure if i should be comparing Forsaken Murderer to Fragment of the Universe.
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE //is the second half of melee damage type. This the armor type checked when attacking someone.
+	stat_attack = CONSCIOUS //originally was HARD_CRIT. This setting makes forsaken murderer attack you until you go into crit.
+	attack_sound = 'sound/effects/hit_kick.ogg' //used chrome to listen to these
+	attack_verb_continuous = "smashes"
+	attack_verb_simple = "smash"
+	faction = NULL //hostile faction makes them friendly with most abnormalities. A choice i dont entirely like.
+	can_breach = TRUE
+	threat_level = TETH_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(60, 60, 50, 50, 50),
+						ABNORMALITY_WORK_INSIGHT = list(40, 40, 30, 30, 30),
+						ABNORMALITY_WORK_ATTACHMENT = list(50, 50, 40, 40, 40),
+						ABNORMALITY_WORK_REPRESSION = list(30, 20, 0, -80, -80),
+						)
+	work_damage_amount = 6
+	work_damage_type = RED_DAMAGE
+	deathmessage = "falls over." //shows in chat when the creature is defeated. Default is "stops moving".
+	environment_smash = 1
+	speak_chance = 2
+	emote_see = list("shakes while mumbling...")
+	emote_hear = list("screams!")
+	max_boxes = 14
+	//success_boxes = 7 //said to be a undefined var
+	zone_selected = BODY_ZONE_CHEST //alternate is BODY_ZONE_HEAD but this made damage go past the armor
+	speed = 5 //Player speed indipendent of Move to Delay.
+
+	ego_list = list(
+		/datum/ego_datum/weapon/regret,
+		/datum/ego_datum/armor/regret
+		)
+
```

Comment:
You have not listed the EGO Gift type in here. We DO have one in the Ego Gift datum, please add this